test/extensions/filters/http/adaptive_concurrency/controller/gradient_controller_test.cc

```diff
@@ -786,6 +791,61 @@ TEST_F(GradientControllerTest, ConsecutiveMinConcurrencyReset) {
   }
 }
 
+// Verify interactions in multi-thread latency samples.
+TEST_F(GradientControllerTest, MultiThreadSampleInteractions) {
```

Comment:
Inspired by the test in https://github.com/envoyproxy/envoy/pull/15004/files#diff-e4b3862c87279e6e5c6253e8f126985dd18a3750546b9d43559fd088b9ce32dfR791

Reply:
hm, typo, updated.